dask/base.py

```diff
@@ -825,10 +825,13 @@ def warn_on_get(get):
     else:
         if get in named_schedulers.values():
             _warnned_on_get[0] = True
-            warnings.warn("The get= keyword has been deprecated. "
-                          "Please use the scheduler= keyword instead with the "
-                          "name of the desired scheduler "
-                          "like 'threads' or 'processes'")
+            warnings.warn(
+                "The get= keyword has been deprecated. "
+                "Please use the scheduler= keyword instead with the name of "
+                "the desired scheduler like 'threads' or 'processes'\n"
+                "    x.compute(scheduler='threads') \n"
+                "or with a function that takes the graph and keys\n"
```

Comment:
Can you explain why this may be needed? I understand the client and named scheduler ones, but re-adding support for get functions seems like a step backwards, as these are meant to be lower-level. They also can't carry additional state around as easily as a named scheduler (lookup in config) or client object (on the object) can.

It was my understanding that @jni's needs would have been satisfied with support only for client objects.

Reply:
There are other cases where people have custom get functions that they want to use.  For example we do this in our tests, so too does xarray.

Here I propose that the `scheduler=` keyword become comfortable accepting a variety of types of objects, including known strings, callables that match the `f(dsk, keys, **kwargs)` signature, Client objects, and whatever future Scheduler object we come up with.  Lets just pass everything through this one keyword and let the `get=` keyword pass away.

Reply:
Ok, makes sense to me. +1.

Reply:
Great.  Thanks for the review.  It's nice to know that you're tracking core changes like these.

Reply:
As a functional programming person, I like the callable interface even better than having to e.g. subclass client, though of course that might end up being the easiest option in many cases. So I very much appreciate the extra flexibility, thanks @mrocklin!